jwst/datamodels/schemas/ifucube.schema.yaml

```diff
@@ -72,4 +76,10 @@ allOf:
       default: 0.0
       ndim: 3
       datatype: float32
+    wavetable:
+      title: Wavelength value for slices
+      fits_hdu: WCS-TAB
+      default: 0.0
+      ndim: 1
+      datatype: float64
```

Comment:
These definitions for the ``wavetable`` object look like they describe a single float64 data value. But this FITS HDU will be a BINTABLE, containing 1 row and at least 1 column containing a 1-D vector of wavelength values. So don't we need definitions for the column name, column data type, etc? (as is done with other data models, like PHOTOM, that include a BINTABLE object)

Reply:
Question. I used the extract1d.schema.yaml as an example because it seemed the closest thing to what I wanted. If it would be better to put it in a table ok. I looked at the photom.schema.yaml
and phometry.schema.yalm and I don't see a BINTABLE object. Maybe I am misreading your comment.
Do I just have to do this is ifucube.schema.yaml
..........
.........
      wavetable:
           title: Wavelength value of IFUCube slices
           fits_hdu: WCS_TABLE
           datatype:
           - name: wavelength
             datatype: float64

Reply:
The extract1d.schema.yaml does in fact include definitions for table column names and data types, e.g.:
```
title: Extracted count rate table
fits_hdu: EXTRACT1D
datatype:
- name: column
  datatype: float64
- name: wavelength
  datatype: float64
- name: background
  datatype: float64
- name: countrate
  datatype: float64
$schema: http://stsci.edu/schemas/fits-schema/fits-schema
```
So following that schema we would define a table containing a single float64 column using:
```
title: Wavelength values
fits_hdu: WCS-TABLE
datatype:
- name: wavelength
  datatype: float64
$schema: http://stsci.edu/schemas/fits-schema/fits-schema
```
The ``ifucube.py`` module also needs updating to include a new entry for the additional table.